models/DataObject/Concrete/Dao/InheritanceHelper.php

```diff
@@ -32,6 +33,8 @@ class InheritanceHelper
 
     const ID_FIELD = 'oo_id';
 
+    const QUERY_ID_FIELD = 'ooo_id';
```

Comment:
Maybe we should name this constant `DEFAULT_QUERY_ID_COLUMN` which is more specific.

Reply:
done